### PR TITLE
Allstar initial setup

### LINKED ISSUE #1: First allstar configuration
## Objective

We need to add the necessary files for the first configuration of `allstar` in the kommiter organization.

## Desscription

It is required to add the following files according to the [allstart configuration setup](https://github.com/ossf/allstar/blob/main/manual-install.md):

* allstar.yaml
* branch_protection.yaml
* binary_artifacts.yaml
* outside.yaml
* security.yaml

And also add a `CHANGELOG` file and improve if necessary the `README`.

### PR DESCRIPTION
**Pull request by CristhianRodriguezMolina**
This PR closes #1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+## 1.0.0 (18.07.2021)
+* First allstar configuration with Opt Out Strategy.
```
Comment (juanhurtado10) [inline CHANGELOG.md]: @CristhianRodriguezMolina @SanGeeky Could you please clarify? What about something like `Allstar config for the kommit's OSPO`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Getting started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* Create a GitHub issue for your contribution, assuming one does not already exist.
+  * Clearly describe the issue including steps to reproduce if it is a bug.
+* Fork the repository on GitHub.
+* Try to keep your local repository in a "rebased" state.
+
+## Finding things to work on
```
Comment (juanhurtado10) [inline CONTRIBUTING.md]: @CristhianRodriguezMolina @SanGeeky This is not really required

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Submitting changes
+
+* All content, comments, pull requests and other contributions must comply with the
+  [Code of Conduct][coc].
+* Push your changes to a topic branch in your fork of the repository.
+* Submit a pull request.
+  * Include a descriptive [commit message][commit-msg].
+  * Changes contributed via pull request should focus on a single issue at a time.
+  * Rebase your local changes against the `main|master` branch. Resolve any conflicts that arise.
```
Comment (juanhurtado10) [inline CONTRIBUTING.md]: @CristhianRodriguezMolina @SanGeeky let's keep just "main". All master branches were renamed.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Additional resources
+
+* Get in touch: [oss@kommit.co](mailto:oss@kommit.co) | [@kommitters_oss](https://twitter.com/kommitters_oss) on Twitter.
+
+[repo]: https://github.com/kommitters/.allstar
+[coc]: https://github.com/kommitters/.allstar/blob/main/CODE_OF_CONDUCT.md
+[commit-msg]: https://github.com/erlang/otp/wiki/Writing-good-commit-messages
+[good-first-issue]: https://github.com/kommitters/.allstar/issues?q=label%3A%22%F0%9F%91%8B+Good+first+issue%22
+[help-wanted]: https://github.com/kommitters/.allstar/issues?q=label%3A%22%F0%9F%86%98+Help+wanted%22
```
Comment (juanhurtado10) [inline CONTRIBUTING.md]: @CristhianRodriguezMolina @SanGeeky please, remove!

File: README.md [hunk 5]
```diff
@@ -1,2 +1,72 @@
...
+`Active repositories: public only`
+
+### Binary artifacts
+
+* No binary artifacts allowed.
+
+`Action: issue`
+`Active repositories: public only`
+
+### Outside collaborators
```
Comment (juanhurtado10) [inline README.md:39]: @CristhianRodriguezMolina @SanGeeky please, split this section out for outside contributors and community members

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (CristhianRodriguezMolina) [reply README.md:39]: @juanhurtado10 this section is about the configurations of the `outside.yaml` policy file, there is not a policy for organization members, so why do you mean by splitting the section?

Comment (juanhurtado10) [reply README.md:39]: @CristhianRodriguezMolina ohhh yes you're right these sections match the Allstar config files. This comment does not make sense. Marked as resolved.

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**